src/rendering/renderers/gl/context/GlContextSystem.ts

```diff
@@ -70,7 +73,12 @@ export class GlContextSystem implements System<ContextSystemOptions>
          * {@link WebGLOptions.powerPreference}
          * @default default
          */
-        powerPreference: 'default',
+        powerPreference: undefined,
+        /**
+         * {@link WebGLOptions.forceFallbackAdapter}
+         * @default false
+         */
+        forceFallbackAdapter: false,
```

Comment:
Hey, after chatting with Mat we decided to remove this setting for webgl since it doesn't make too much sense

```suggestion
```

Reply:
Makes sense, I thought we might end up doing that but left it in for final decision. Have removed.